oidc-fapi-conformance-tests/test_runner_fapi.sh

```diff
@@ -0,0 +1,53 @@
+
+echo "========================"
+echo "Running Tests"
+echo "========================"
+
+echo
+echo "FAPI Basic test plan - private_key_jwt, plain_fapi, plain_response, by_value"
+echo "-----------------------------"
+echo
+python3 $CONFORMANCE_SUITE_PATH/scripts/run-test-plan.py fapi1-advanced-final-test-plan[client_auth_type=private_key_jwt][fapi_profile=plain_fapi][fapi_response_mode=plain_response][fapi_auth_request_method=by_value] $PATH_TO_SCRIPTS/config/IS_config_fapi_pvtkeyjwt.json 2>&1 | tee fapi-testplan-pvtkeyjwt-log.txt
+echo
+
+echo
+echo "FAPI Basic test plan - mtls, plain_fapi, plain_response, by_value"
+echo "-----------------------------"
+echo
+python3 $CONFORMANCE_SUITE_PATH/scripts/run-test-plan.py fapi1-advanced-final-test-plan[client_auth_type=mtls][fapi_profile=plain_fapi][fapi_response_mode=plain_response][fapi_auth_request_method=by_value] $PATH_TO_SCRIPTS/config/IS_config_fapi_mtls.json 2>&1 | tee fapi-testplan-mtls-log.txt
+echo
+
+# echo
+# echo "FAPI Basic test plan - private_key_jwt, plain_fapi, plain_response, pushed"
+# echo "-----------------------------"
+# echo
+# python3 $CONFORMANCE_SUITE_PATH/scripts/run-test-plan.py fapi1-advanced-final-test-plan[client_auth_type=private_key_jwt][fapi_profile=plain_fapi][fapi_response_mode=plain_response][fapi_auth_request_method=pushed] $PATH_TO_SCRIPTS/config/IS_config_fapi_pvtkeyjwt_par.json 2>&1 | tee fapi-testplan-pvtkeyjwt-par-log.txt
+# echo
+
+# echo
+# echo "FAPI Basic test plan - mtls, plain_fapi, plain_response, pushed"
+# echo "-----------------------------"
+# echo
+# python3 $CONFORMANCE_SUITE_PATH/scripts/run-test-plan.py fapi1-advanced-final-test-plan[client_auth_type=mtls][fapi_profile=plain_fapi][fapi_response_mode=plain_response][fapi_auth_request_method=pushed] $PATH_TO_SCRIPTS/config/IS_config_fapi_mtls_par.json 2>&1 | tee fapi-testplan-mtls-par-log.txt
+# echo
+
+# echo
+# echo "FAPI Basic test plan - private_key_jwt, plain_fapi, jarm, by_value"
+# echo "-----------------------------"
+# echo
+# python3 $CONFORMANCE_SUITE_PATH/scripts/run-test-plan.py fapi1-advanced-final-test-plan[client_auth_type=private_key_jwt][fapi_profile=plain_fapi][fapi_response_mode=jarm][fapi_auth_request_method=by_value] $PATH_TO_SCRIPTS/config/IS_config_fapi_pvtkeyjwt.json 2>&1 | tee fapi-testplan-pvtkeyjwt-jarm-log.txt
+# echo
+
+# echo
+# echo "FAPI Basic test plan - mtls, plain_fapi, jarm, by_value"
+# echo "-----------------------------"
+# echo
+# python3 $CONFORMANCE_SUITE_PATH/scripts/run-test-plan.py fapi1-advanced-final-test-plan[client_auth_type=mtls][fapi_profile=plain_fapi][fapi_response_mode=jarm][fapi_auth_request_method=by_value] $PATH_TO_SCRIPTS/config/IS_config_fapi_mtls.json 2>&1 | tee fapi-testplan-mtls-jarm-log.txt
+# echo
```

Comment:
```suggestion
# echo

```

Reply:
Fixed with [5655e1a](https://github.com/wso2/product-is/pull/18059/commits/5655e1a6d406c0596f517b3e262483b0a3032be7)